### PR TITLE
Feature/apis catalog pagination

### PR DESCRIPTION
**Pull request by marla-singer**
Part of #115 
- Implemet pagination in APIs catalog via package
- Setting sorting & filtering
### Changes
- Remove setQueryParams from template logic (for APIs catalog)
- Add setQueryParams on routing (for APIs catalog)

### CODE REVIEW

File: users/client/lib/router.js [hunk 1]
```diff
@@ -47,15 +47,12 @@ FlowRouter.route('/settings/profile', {
...
-  action: function () {
-    BlazeLayout.render('masterLayout', { main: 'signOut' });
-  },
+  triggersEnter: [
+    function () {
+      Meteor.logout(() => {
+        FlowRouter.go('catalogue');
+      });
+      // Page render
+      BlazeLayout.render('masterLayout', { main: undefined });
```
Comment (jykae) [inline users/client/lib/router.js]: @marla-singer what that undefined does? is it bug? 🤔

Comment (jykae) [inline users/client/lib/router.js]: Looks like it works, could you make better comment for it?

Comment (jykae) [inline users/client/lib/router.js]: @marla-singer ok, could you add that "force render" explanation to comment somehow short way as it is not obvious what it does. Eg. "Use undefined to force page render when route is not changed"

**Comment by jykae** [comment]
@marla-singer Overall good work. 👍

**Comment by marla-singer** [comment]
@jykae Done :)

**Pull request merged**